### ISSUE TITLE
false positiv on some PDF files

### ISSUE THREAD

**Issue by BugBuster1701** [phpMussel/phpMussel#77]
Some, not all,  PDFs make problems.

```
Detected phpMussel-PHP.Packer.RX-00 (066DLM_Starfleet_Diploma.pdf)!
Detected phpMussel-HEUR.PDF.Exploit.1505X.A (Contao-Symfony-Timon-Schroeter.pdf)!
```

```
Thu, 19 Nov 2015 19:43:43 +0100 Started.
> Checking '066DLM_Starfleet_Diploma.pdf' (FN: c71a1232; FD: 45f2bf79):
-> Detected phpMussel-PHP.Packer.RX-00!
Thu, 19 Nov 2015 19:43:46 +0100 Finished.

Thu, 19 Nov 2015 19:54:57 +0100 Started.
> Checking 'Contao-Symfony-Timon-Schroeter.pdf' (FN: 286f271a; FD: 42d5cd5c):
-> Detected phpMussel-HEUR.PDF.Exploit.1505X.A!
Thu, 19 Nov 2015 19:55:03 +0100 Finished.
```

**Comment by BugBuster1701** [comment]
ZIP with PDFs
http://goo.gl/gRNoHi

**Comment by Maikuolan** [comment]
I'd originally written the "phpMussel-HEUR.PDF.Exploit.1505X.A" signature due to some information that was sent to me suggesting a particular type of PDF vulnerability relating to a particular type of included image, but, I looked into it again just now, and it seems the information was bad. The signature was targeting the "Ducky" tag included in the comments field of images included in PDF files, but, it seems this tag is actually a normal tag used by images produced by some Adobe applications, and is completely benign. This signature has been removed as of the latest commit.

The "phpMussel-PHP.Packer.RX-00" signature should only be targeting PHP files, so, to trigger on a PDF file, is a false positive. As of the latest commit, I've included a stricter condition to hopefully ensure that it only targets PHP files.

Thanks for spotting these. :-)
